frontend/src/metabase/querying/notebook/components/DataStep/tests/setup.tsx

```diff
@@ -6,19 +6,26 @@ import {
 } from "__support__/server-mocks";
 import { renderWithProviders } from "__support__/ui";
 import { createMockModelResult } from "metabase/browse/models/test-utils";
+import * as envs from "metabase/env";
 import * as Lib from "metabase-lib";
 import { columnFinder } from "metabase-lib/test-helpers";
 import { createSampleDatabase } from "metabase-types/api/mocks/presets";
-import {
-  createMockEmbedState,
-  createMockState,
-} from "metabase-types/store/mocks";
 
 import { createMockNotebookStep } from "../../../test-utils";
 import type { NotebookStep } from "../../../types";
 import { NotebookProvider } from "../../Notebook/context";
 import { DataStep } from "../DataStep";
 
+const mockedEnvs = envs as { isEmbeddingSdk: boolean };
+
+jest.mock("metabase/env", () => {
```

Comment:
This mocks the file we read `process.env.IS_EMBEDDING_SDK`, since we'll be using that instead of `isEmbeddingSdk` inside redux from now on.